some packaging files

necrose99 opened the pull request:
some packaging files 

rpm spec skell 

debian skell , ie python3-maryam prepackaged.. /stdeb auto.. 

MSI file if you care to add to releases... 


gitignore add more ignore directories for  **py* setup.py bdist_wheel** or etc to not get unwanted leftovers.. 
ie bdist_msi bdist_rpm etc... 


testing my gentoo ebuild for @pentoo 


pipefile you can regen for devsecops/vultest code...

necrose99 committed:
update gitignore

add pkg skells

necrose99 committed:
pipefiles

necrose99 committed:
Merge https://github.com/saeeddhqan/Maryam

saeeddhqan commented on Pipfile:
```diff
@@ -0,0 +1,19 @@
+    bs4
+    cloudscraper
+commands =
+    python setup.py test
```
@necrose99 currently, we don't have test files.

necrose99 replied:
centos 8 / debian 9 live in vmware , vmw-fs to folder ...  

your welcome to add to a branch as templates , for now.. 

my next task is to test ebuild for Gentoo/@pentoo 

pull to pentoo

saeeddhqan commented on tox.ini:
```diff
@@ -0,0 +1,17 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
```
currently, we don't have test files.

necrose99 committed:
cp items from Makefile over

# Testing

check:  ## Check linting
	flake8
	isort . --project maryam --check-only --diff
	black . --check


fmt:  ## Format source
	isort . --project maryam
	black .

necrose99 committed:
Update tox.ini

add items from makefile ie flake 8 etc...

necrose99 committed:
Update .gitignore

necrose99 committed:
Update Pipfile

necrose99 committed:
Update .gitignore

saeeddhqan commented:
gitignore is good.
why should we have Debian files and tox? just because of pentoo?

necrose99 commented:
https://salsa.debian.org/necrose99/maryam/  **chown to do (ie change owner )** 
 , anyways can rig docker to continually deb this / mirror repo.. and done  for debian  ie apt install python3-maryam 
rpm spec can likewise be offered unto that community... 

hopefully lures in more devs....

 i switched  this tox basicly mirrors **make test** ...  it will report on compat  with newer pythons automatically hopefully. 
can pull to another branch and burn off unwanteds.. 
tox makes @gentoo devs possibly qa happy ... if pentoo package moves over to gentoo someday.. 

as for pentoo...  ive added a pull.

saeeddhqan commented:
@necrose99 Do you have an email address? or slack?

saeeddhqan commented:
If you need any change, you can do it. you can have access to the repository.
